doc/classes/Object.xml

```diff
@@ -129,12 +129,12 @@
 				Calls the [code]method[/code] on the object and returns the result. Contrarily to [method call], this method does not support a variable number of arguments but expects all parameters to be via a single [Array].
 				[codeblocks]
 				[gdscript]
-				var node = Node2D.new()
-				node.callv("set", ["position", Vector2(42, 0)])
+				var node = Node3D.new()
+				node.callv("rotate", [Vector3(1.0, 0.0, 0.0), 1.571])
 				[/gdscript]
 				[csharp]
-				var node = new Node2D();
-				node.Callv("set", new Godot.Collections.Array { "position", new Vector2(42, 0) });
+				var node = new Node3D();
+				node.Callv("rotate", new Godot.Collections.Array { new Vector2(1.0, 0.0, 0.0), 1.571 });
```

Comment:
```suggestion
				node.Callv("rotate", new Godot.Collections.Array { new Vector3(1f, 0f, 0f), 1.571f });
```

Reply:
...and I forgot to change one of the `Vector2` to a `Vector3`